README.md

```diff
@@ -34,7 +34,7 @@ config :power_control,
 
 ## Benchmarks
 
-Lets take a look at the following fibonacci benchmark run on a Raspberry Pi Zero W:
+Lets take a look at the following fibonacci benchmark run on a Raspberry Pi Zero W using different [CPU scaling governors](https://wiki.archlinux.org/index.php/CPU_frequency_scaling#Scaling_governors):
```

Comment:
https://www.kernel.org/doc/html/v4.15/admin-guide/pm/cpufreq.html would be a good link here, could say something such as "using different [`CPUFreq`](https://www.kernel.org/doc/html/v4.15/admin-guide/pm/cpufreq.html) settings"